<?xml version="1.0"?>

<launch>
  <arg name="file_out"  default="benchmarkfile"/>

  <include file="$(find ucl_drone)/launch/slam.xml" />
  <include file="$(find ucl_drone)/launch/global_params.xml" />
  <include file="$(find ucl_drone)/launch/gui.xml" />
  <include file="$(find ucl_drone)/launch/driver.xml" />
  <node name="ucl_drone_manual_pose_estimation" pkg="ucl_drone" type="manual_pose_estimation" output="screen"></node>

  <node pkg="rosbag" type="record" name="recorder" output="screen" args="-O /home/$(env USER)/bagfiles/$(arg file_out).bag /ardrone/front/image_rect_color /manual_pose_estimation"/>
</launch>
